src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/navbar-logo.html

```diff
@@ -7,17 +7,11 @@
 {% endif %}
 <a class="navbar-brand logo" href="{{ href }}">
   {# get all the brand information from html_theme_option #}
-  {% set is_logo = logo_url or theme_logo.get("image_light") or theme_logo.get("image_dark") %}
-  {% set image_light = theme_logo.get("image_light") or logo_url %}
-  {% set image_dark = theme_logo.get("image_dark") or logo_url %}
-  {% set image_light = image_light if image_light.startswith("http") else pathto('_static/' + image_light, 1) %}
-  {% set image_dark = image_dark if image_dark.startswith("http") else pathto('_static/' + image_dark, 1) %}
+  {% set is_logo = theme_logo.get("logo")["light"] != "" %}
   {% set alt = theme_logo.get("alt_text", "Logo image") %}
   {% if is_logo %}
-    <img src="{{ image_light }}"
-         class="logo__image only-light"
-         alt="{{ alt }}"/>
-    <img src="{{ image_dark }}" class="logo__image only-dark" alt="{{ alt }}"/>
+    <img src="{{ theme_logo.get('logo')['light'] }}" class="logo__image only-light" alt="{{ alt }}"/>
+    <img src="{{ theme_logo.get('logo')['dark'] }}" class="logo__image only-dark" alt="{{ alt }}"/>
```

Comment:
just a curiosity question: is there a reason you mix `.get(key)` and `[key]` types of access here? (and above on line 10)

Reply:
For a reason I can't explain, `.get('light')` doesn't work, this implementation does....